source/gent/tier1_hortense.rst

```diff
@@ -52,6 +52,13 @@ Hortense consists of the following partitions:
        - 4x NVIDIA A100-SXM4 (80 GB GPU memory), NVLink3
        - 512 GiB RAM (~10GB/CPU core), no swap
        - 480 GB SSD local disk
+- ``dodrio/debug_rome``: interactive and debug partition:
+   - 3 virtual workernodes, each with:
```

Comment:
What's "virtual" about these workernodes? Are they VMs?

I would drop the `virtual`, it'll only lead to confusion.

Reply:
ok, but these are indeed VMs